scss/inc/_functions.scss

```diff
@@ -270,3 +270,21 @@ Usage:
 @mixin disableSelect() {
     @include vendor-prefix(user-select, none, property);
 }
+
+/*
+ * Define for left to right text direction rules
+ */
+@mixin ltr {
+    body:not([dir="rtl"]) & {
```

Comment:
I still need to check locally, but be aware that there can be set a `dir` on the body and on the item level
![image](https://user-images.githubusercontent.com/11692751/126511002-a1d53411-bf22-447b-b6d8-4e6bf6fd5b8a.png)

Reply:
what is idea behind? to change item and the UI independently?

Reply:
According previous ticket https://github.com/oat-sa/tao-item-runner-qti-fe/pull/200/files#diff-8d43b4e4068f3999fa3b0c9a0c778e60b3f8981741849530833d6b7acdb47f57R88
`dir` will be on item body, not on document body